src/main/java/com/redhat/pantheon/servlet/AsciidocRenderingServlet.java

```diff
@@ -44,40 +43,25 @@
 import java.io.IOException;
 import java.io.Writer;
 import java.util.HashMap;
-import java.util.List;
 import java.util.Map;
-import java.util.stream.Collectors;
 
 import static java.util.Collections.singletonList;
 import static java.util.stream.Collectors.toList;
 
-/**
- * Hello World Servlet registered by resource type
- *
- * Annotations below are short version of:
- * 
- * @Component
- * @Service(Servlet.class)
- * @Properties({
- *    @Property(name="service.description", value="Hello World Type Servlet"),
- *    @Property(name="service.vendor", value="The Apache Software Foundation"),
- *    @Property(name="sling.servlet.resourceTypes", value="sling/servlet/default"),
- *    @Property(name="sling.servlet.selectors", value="hello"),
- *    @Property(name="sling.servlet.extensions", value="html")
- * })
- */
-@Component(
-        service = Servlet.class,
-        property = {
-                "sling.servlet.resourceTypes=pantheon/modules",
-                "sling.servlet.extensions=preview",
-                Constants.SERVICE_DESCRIPTION+"=Servlet which transforms asciidoc content into html",
-                Constants.SERVICE_VENDOR+"=Red Hat Content Tooling team"
-        })
+
+@Component(service = { Servlet.class })
+@SlingServletResourceTypes(
+        resourceTypes="pantheon/modules",
+        methods= "GET",
+        extensions="preview")
 @SuppressWarnings("serial")
 public class AsciidocRenderingServlet extends SlingSafeMethodsServlet {
+
+    private static final String ADOC_NODE_NAME = "asciidoc";
+    private static final String CONTENT_NODE_NAME = "jcr:content";
+    private static final String CACHE_NODE_NAME = "cachedContent";
     
-    private final Logger log = LoggerFactory.getLogger(AsciidocRenderingServlet.class);
+    private final Logger log = LoggerFactory.getLogger("com.redhat");
```

Comment:
Why the change to a package name? The class name gives you far more specificity on the log entries.

Reply:
My logic was that during the course of development, I may need to destroy my sling settings folder as part of each deployment, including loggers I have set up. Rather than needing to re-set up 5 or so loggers each time, it's easier to just do 1.

Having said that, I haven't actually had that problem yet, and I'm also a big fan of "don't solve problems you don't have and may never" so, we can put it back the way it was.

Reply:
I'd agree with that.

Reply:
```suggestion
    private final Logger log = LoggerFactory.getLogger(AsciidocRenderingServlet.class);
```

Reply:
:+1: